.github/workflows/ci.yml

```diff
@@ -69,6 +69,7 @@ jobs:
       - name: Run tests
         shell: bash -l {0}
         run: |
+          cd ..
           pip install -e KinFragLib
```

Comment:
Good catch - sorry that was my bad!
 
To avoid changing directories it _should_ be possible to replace 

```bash
cd ..
pip install -e KinFragLib
```
with
```bash
python setup.py install
```

Reply:
Perfect, I will try it this way :)

Reply:
Did not work, changed it back to the cd .. and pip install version and added the comment.